components/arm/pb_helpers.go

```diff
@@ -4,11 +4,53 @@ import (
 	"fmt"
 
 	commonpb "go.viam.com/api/common/v1"
+	pb "go.viam.com/api/component/arm/v1"
 
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/referenceframe/urdf"
+	"go.viam.com/rdk/utils"
 )
 
+const (
+	defaultMaxVelDegPerSec  = 60.
+	defaultMaxAccDegPerSec2 = 1.
+)
+
+type MoveOptions struct {
+	MaxVel, MaxAcc float64
+}
+
+func NewDefaultMoveOptions() *MoveOptions {
+	return moveOptionsFromProtobuf(nil)
+}
+
+func moveOptionsFromProtobuf(protobuf *pb.MoveOptions) *MoveOptions {
+	if protobuf == nil {
+		protobuf = &pb.MoveOptions{}
+	}
+
+	var vel, acc float64
+	if protobuf.MaxVelDegsPerSec == nil {
+		vel = defaultMaxVelDegPerSec
+	}
+	if protobuf.MaxAccDegsPerSec2 == nil {
+		acc = defaultMaxAccDegPerSec2
+	}
+	return &MoveOptions{
+		MaxVel: utils.DegToRad(vel),
+		MaxAcc: utils.DegToRad(acc),
+	}
+}
+
+func (opts MoveOptions) toProtobuf() *pb.MoveOptions {
+	vel := utils.RadToDeg(opts.MaxVel)
```

Comment:
shouldn't this be handled on the driver level?, i.e. converting from rad to deg?

Reply:
No. This should be done ASAP so that the arm interface is getting the sanitized version and everything is in radians

Reply:
I am confused, isn't this converting from radians to degrees?

Reply:
Yes.  Its following the pattern we have done throughout the stack of defining an RDK struct and writing "to" and "from" conversion functions to convert it back to protobuf.

Reply:
There is a complementary function that converts to radians

Reply:
> Its following the pattern we have done throughout the stack of defining an RDK struct and writing "to" and "from" conversion functions to convert it back to protobuf.

This makes sense to me -- the conversion of fromProto and then toProto.

> There is a complementary function that converts to radians

sorry, I'm just stuck on understandings why we'd want to accept a unit in degs, convert it to rad, and then convert it back to degs. That seems like it adds unnecessary complexity. 
Perhaps I am just being silly with reading through the changes made in this PR and I am just totally missing something. Would you mind walking me through?

Reply:
Everything we operate on should be in terms of radians for consistency.  The proto representation is explicitly in degrees.  While it is in proto form it needs to be in degrees, which is what we are returning.  While it is in RDK form it needs to be in radians

Reply:
> Everything we operate on should be in terms of radians for consistency. The proto representation is explicitly in degrees. 

Ok this finally makes sense to me. The source of my confusion is then not understanding why the proto representation is in degrees -- is just a design choice?